xtask/src/main.rs

```diff
@@ -0,0 +1,329 @@
+use std::{fmt::Debug, io, process::Output, vec};
+
+use cargo_metadata::MetadataCommand;
+use clap::{Parser, Subcommand, ValueEnum};
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+use color_eyre::{eyre::Context, Result};
+use duct::cmd;
+use tracing::level_filters::LevelFilter;
+use tracing_log::AsTrace;
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+    let args = Args::parse();
+    tracing_subscriber::fmt()
+        .with_max_level(args.log_level())
+        .without_time()
+        .init();
+
+    match args.run() {
+        Ok(_) => (),
+        Err(err) => {
+            tracing::error!("{err}");
+            std::process::exit(1);
+        }
+    }
```

Comment:
I originally had this as just falling through and letting color_eyre printt the error, but changes to this for a little nicer output. It may be worth considering changing back at some point.